### PR TITLE
Civ 2814 get event token

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2814

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by laarmada — APPROVED** [review]

**Review by taomoh — COMMENTED** [review]
Looking good. I only have one comment about URL pattern

File: src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java [hunk 1]
```diff
@@ -106,4 +110,17 @@ public ResponseEntity<SearchResult> getCaseList(@RequestHeader(HttpHeaders.AUTHO
         List<DashboardClaimInfo> ocmcClaims = claimStoreService.getClaimsForDefendant(authorization, submitterId);
         return new ResponseEntity<>(ocmcClaims, HttpStatus.OK);
     }
+
+    @GetMapping(path = "/defendant/{submitterId}/response/submit/{caseId}/token")
```
[the remark this answers is not in the document]
Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java:114]: I like this

/defendants/{submitterId}/case-responses/{caseId}/event-token

**Review by taomoh — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**